src/consumer/component/sources/dist.js

```diff
@@ -53,7 +51,7 @@ export default class Dist extends Source {
 
   buildSrcWithSourceMapAnnotation(fileName: string) {
     return this.mappings ?
-    `${this.src}\n\n//# sourceMappingURL=${fileName}${MAP_EXTENSION}` : this.src;
+    `${this.compiledContent}\n\n//# sourceMappingURL=${fileName}${MAP_EXTENSION}` : this.compiledContent;
```

Comment:
I think this should be this.contents instead of compiledContent (both instances in this line)